pkg/observer/data_stats.go

```diff
@@ -13,6 +13,13 @@ var (
 		Help:        "Data event statistics. For internal use only.",
 		ConstLabels: nil,
 	}, []string{"event"})
+
+	DataEventSizeHist = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:        consts.MetricNamePrefix + "data_event_size_histogram",
```

Comment:
I switched all metrics to use a `Namespace` field instead of concatenating the prefix in #1228, can you change this one too?